components/atoms/EntitlementBox.js

```diff
@@ -2,7 +2,10 @@ import PropTypes from "prop-types";
 
 export function EntitlementBox(props) {
```

Comment:
Entitlement.stories.js is missing arguments to display in the storybook.

Look at DrawerItem.stories.js for an example.